doc/generic/pgf/text-en/pgfmanual-en-base-images.tex

```diff
@@ -66,7 +66,7 @@ \subsection{Declaring an Image}
 \begin{command}{\pgfdeclareimage\oarg{options}\marg{image name}\marg{filename}}
     Declares an image, but does not paint anything. To draw the image, use
     |\pgfuseimage{|\meta{image name}|}|. The \meta{filename} may not have an
-    extension.  For \textsc{pdf}, the extensions |.pdf|, |.jpg|, and |.png|
+    extension. For \textsc{pdf}, the extensions |.pdf|, |.jpg|, and |.png|
```

Comment:
In the stated Wikipedia article there isn't stated that this is the correct way to space sentences. Rather there seems to be opposing opinions and for the newer age it seems to tend to using on space. But that is not the point here because in LaTeX we can decide what we want to see in the output and has nothing to do with the spaces in the input. (I think we agree on that.)

But of course that **editors** use them is a very relevant point, especially when the main contributor to this project uses one of them ;)

I know that you stated that point in the past already and we agreed that I will not touch any of the "two spaces" you have written (for the above reason). And I am pretty sure that I haven't.

So just for clarity: If you insist to revert that changes I will do it. But I just want to mention *my* point again which is *consistency*. In the given example above there are two more sentences before the one that is diffed and there is only given 1 space after the end-of-sentence-dot.

So could you live with the current pull request or shall I change it as you wish? (Please note that this was a one-time-search for the two spaces and thus will not happen again.)

Reply:
Any news here?

Reply:
I was waiting for your answer which is still lacking. But I'll take it as "you still wish that I revert the end-of-sentence-dots".
[Below](https://github.com/pgf-tikz/pgf/pull/886#issuecomment-650497371) I asked another question which is also not answered yet. Could you please let me know your opinion, too, so I don't do unnecessary/double work. Thank you very much in advance.